composer/callbacks/checkpoint_saver.py

```diff
@@ -423,10 +440,10 @@ def _rotate_checkpoints(self, sharding_enabled: bool = False):
 
         while len(self.saved_checkpoints) > self.num_checkpoints_to_keep:
             prefix_dir = None
-            checkpoint = self.saved_checkpoints.pop(0)
-            prefix_dir = str(Path(checkpoint).parent)
+            checkpoint_to_delete = self.saved_checkpoints.pop(0)
```

Comment:
nit: was raising pylint warnings that we could be overriding the `checkpoint` variable (import of `composer/utils/checkpoint.py`)